LevelWorld/LevelWindow.cpp

```diff
@@ -32,9 +39,11 @@ HelloWindow::HelloWindow(BRect frame)
 	AddChild(aView);
 
 	BMenu* fileMenu = new BMenu("Level");
-	BMenuItem* loadMenu = new BMenuItem("Open", NULL, 'O');
-	BMenuItem* saveMenu = new BMenuItem("Save", new BMessage('SAVE'), 'S');
-	BMenuItem* saveAsMenu = new BMenuItem("Save As" B_UTF8_ELLIPSIS, NULL, 'S', B_SHIFT_KEY);
+	BMenuItem* loadMenu = new BMenuItem("Open",
```

Comment:
Add a B_UTF8_ELLIPSIS after "Open". It opens the file panel. Whenever a new window is opened, we use ellipses.